linux-entra-sso.py

```diff
@@ -80,6 +80,7 @@ def __init__(self, daemon=False):
         self.broker = None
         self.session_id = uuid.uuid4()
         self._state_changed_cb = None
+        self._last_state_reported = False
```

Comment:
For whatever reason this change results in a loss of the first "broker offline" notification. To reproduce, do the following:

1. start broker (ensure that it is running)
2. start browser (you should be logged in)
3. stop broker (the extension is not notified about the state change)
4. start broker again (the extension gets the "connection to broker restored" event)
5. stop broker again (the extension gets the "lost connection to broker" event).

Reply:
Does the script get the disconnect message? I've seen losses on the bus as well, not caused by the script.

Anyway, let me try to reproduce.

Reply:
At least the notify callback that notifies the browser part is not called.

Reply:
Fixed: We needed to track the last state unconditionally as the callback is not available in the beginning.

Reply:
Yep. Now it works. Thanks for fixing.